1.0.1/start

```diff
@@ -5,7 +5,13 @@ export HSQL_SERVER_PORT=$HSQLDB_PORT_9101_TCP_PORT
 
 export ZK_CLIENT_CONNECT=$ZOOKEEPER_PORT_2181_TCP_ADDR:$ZOOKEEPER_PORT_2181_TCP_PORT
 
-export SPRING_REDIS_HOST=$REDIS_PORT_6379_TCP_ADDR
-export SPRING_REDIS_PORT=$REDIS_PORT_6379_TCP_PORT
+if [ -n "$REDIS_PORT" ]; then
```

Comment:
I'd rather drive the decision on the value of `XD_TRANSPORT` rather than `REDIS_PORT`, what do you think?